crates/typed-store/tests/macro_tests.rs

```diff
@@ -124,6 +129,90 @@ async fn macro_test() {
     assert_eq!(format!("\"8\""), *m.get(&"\"8\"".to_string()).unwrap());
 }
 
+#[derive(SallyDB)]
+pub struct SallyDBExample {
+    col1: SallyColumn<String, String>,
+    col2: SallyColumn<i32, String>,
+}
+
+#[tokio::test]
+async fn macro_test_sallydb() {
```

Comment:
why is this called macro test?

Reply:
ah it is just how other tests were named in this test module, i'll name it better

Reply:
fixed